Packs/MongoDBAtlas/ModelingRules/MongoDBAtlas/MongoDBAtlas.xif

```diff
@@ -0,0 +1,41 @@
+[MODEL: dataset="mongodb_atlas_raw"]
+filter source_log_type = "alerts"
+| alter
+	xdm.event.type = source_log_type,
+	xdm.alert.description = acknowledgementComment,
+	xdm.source.user.username = acknowledgingUsername,
```

Comment:
I'm not sure this is the field I would map it to - To my understanding, Acknowledging Username is the username of the user who worked on the alert on the SOC side, while the source.user.username is likely associated with the username that the alert was triggered on